config/samples/lws_v1alpha1_configuration.yaml

```diff
@@ -1,12 +1,33 @@
 apiVersion: lws.x-k8s.io/v1alpha1
 kind: Configuration
 metadata:
-  labels:
-    app.kubernetes.io/name: configuration
-    app.kubernetes.io/instance: configuration-sample
-    app.kubernetes.io/part-of: lws
-    app.kubernetes.io/managed-by: kustomize
-    app.kubernetes.io/created-by: lws
   name: configuration-sample
 spec:
-  # TODO(user): Add fields here
+  ## Examples as below (with their default values):
+  ## refer to `Configuration` struct definition for more details.
+  #
+  # webhook:
+  #   port: 9443
+  #   host: ""
+  #   certDir: "/tmp/k8s-webhook-server/serving-certs"
+  #
+  # leaderElection:
+  #   leaderElect: true
+  #   # ... other election parameters ..
```

Comment:
This prevents the functionality of copying and pasting of this example, can we remove?.